### DISCUSSION TITLE
Studio SQL Editor:  Files being overwritten

### DISCUSSION THREAD

**Discussion by angus-clark** [supabase/supabase#7922] [Questions]
When I logged into the Supabase Studio I found that my sql editor had overwritten my create tables script with the contents of another script I had created. I've tried refreshing but no luck. Is there a fix to this? Has anyone else experienced this?

**Comment by silentworks** [comment]
Are you referring to the local development environment for Supabase?

**Comment by angus-clark** [comment]
I am referring to the online studio/dashboard SQL editor.

Angus
